pkg/agent/baker.go

```diff
@@ -826,6 +826,19 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 			}
 			return 0
 		},
+		"ShouldConfigureCustomCATrust": func() bool {
+			return config.CustomCATrustConfig != nil && len(config.CustomCATrustConfig.CustomCATrustCerts) > 0
+		},
+		"GetCustomCATrustConfigCerts": func() []string {
+			if config.CustomCATrustConfig != nil && config.CustomCATrustConfig.CustomCATrustCerts != nil {
+				indentedCerts := make([]string, 0, len(config.CustomCATrustConfig.CustomCATrustCerts))
+				for _, cert := range config.CustomCATrustConfig.CustomCATrustCerts {
+					indentedCerts = append(indentedCerts, datamodel.IndentString(cert, 4))
+				}
+				return config.CustomCATrustConfig.CustomCATrustCerts
```

Comment:
I might be missing something, but do you mean to be returning indentedCerts here?

Reply:
ironically the output looks good in the generated yaml so I think the indentation may not be necessary?

Reply:
Yeah just tested, i think it really is not needed. Will remove indentation